neuro_data/movies/data_schemas.py

```diff
@@ -818,7 +818,7 @@ def __getattr__(self, item):
         if not item in self.h5_handle[self.name]:
             raise AttributeError('{} is not among the attributes'.format(item))
         else:
-            ret = self.h5_handle[self.name][item].value
+            ret = self.h5_handle[self.name][item][()]
```

Comment:
I think the more standard way to do that is [:]. Also I think calling `np.array(self.h5_handle[self.name][item])` is probably more explicit (what you are doing is transforming an h5py dataset to a numpy array). Up to you, though, not major